HumanControls/HumanControls.ino

```diff
@@ -38,11 +38,9 @@
 
 //Joystick deadzones
 #define X_DEAD_ZONE_SIZE 100 //Total size of the x deadzone
-#define X_MIN 0
-#define X_MAX 1023
+#define MIN 0                //Minimum joystick value that comes from the sensor
```

Comment:
These should probably be `JOYSTICK_MIN` and `JOYSTICK_MAX`

Reply:
Done